### PR TITLE
New DevTools experiment page: Edge Command Palette

### PR DESCRIPTION
**Pull request by captainbrosset**
**Edge Command Palette** is a new experiment that makes it possible to run useful productivity and developer commands in Edge by pressing `Ctrl+Shift+Space`. This includes running DevTools commands without having to open DevTools first.

The goal of this PR is to document this experiment so that people know how to enable it and use it.

Rendered page: https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette?branch=pr-en-us-2023

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Streamline your workflows with Edge Command Palette
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:2]: consider concrete specific title eg:
Run browser and DevTools commands via keyboard with Edge Command Palette

**Review by mikehoffms — APPROVED** [review]

**Comment by captainbrosset** [comment]
Let's wait before we merge this, ideally we'd want string changes to be done in the code first, so that "Toaster" doesn't appear in the about:flags page.

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+## Enable Command Palette
+
+By default, Command Palette isn't enabled. To enable the Command Palette experiment:
+
+1. In Microsoft Edge, go to `edge://version`, and make sure you're using Microsoft Edge 105 or later.  To get the latest preview channels of Microsoft Edge, see [Microsoft Edge Insider Channels](https://www.microsoftedgeinsider.com/en-us/download/).
+
+1. Go to `edge://flags`.
+
+<!-- DevTools Toaster is in the process of being changed in Edge to match the feature's name -->
+1. In the **Search flags** text field, type **DevTools Toaster** or **Command Palette**.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:29]:
```suggestion
1. In the **Search flags** text field, type **Command Palette**.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+<!-- DevTools Toaster is in the process of being changed in Edge to match the feature's name -->
+1. In the **DevTools Toaster** or **Command Palette** dropdown menu, select **Enabled**:
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:32]:
```suggestion
1. In the **Search flags** text field, type **Command Palette**.

1. In the **Command Palette** dropdown menu, select **Enabled**:
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+   ![Enabling the Command Palette flag in the edge://flags page](./media/command-palette-flag.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:34]: update screenshot

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:32]: do this update in a later PR

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:34]: do this update in a later PR

**Pull request merged**